js/core/form/editors/DateTimeEditorView.js

```diff
@@ -15,7 +15,8 @@ import DateView from './impl/dateTime/views/DateView';
 import TimeView from './impl/dateTime/views/TimeView';
 
 const defaultOptions = {
-    enableDelete: false
+    enableDelete: false,
+    timezoneOffset: 180 // minutes
```

Comment:
The default value should be `null` which should mean 'take the browser timezone'.